Sources/muterCore/CLICommands/MuterError.swift

```diff
@@ -5,6 +5,7 @@ public enum MuterError: Error, Equatable {
     case projectCopyFailed(reason: String)
     case unableToCreateSwapFileDirectory(reason: String)
     case noSourceFilesDiscovered
+    case noSourceFilesOnExclusiveList
```

Comment:
i'd love to change this name. `noSourceFilesDiscoveredFromMutateList` is really descriptive but extremely long.

Reply:
`noSourceFileToMutate`?

Reply:
`noSourceFilesToMutateFound`?